lib/modules/platform/gitea/pr-cache.ts

```diff
@@ -150,7 +159,7 @@ export class GiteaPrCache {
       }
 
       const uri = parseUrl(parseLinkHeader(res.headers.link)?.next?.url);
-      url = uri ? `${uri.pathname}${uri.search}` : undefined;
+      query = uri ? uri.search : undefined;
```

Comment:
```suggestion
      query = uri ? uri.search : null;
```
We should not set ourselfs `undefined`